2_process/src/write_csv_fcn.R

```diff
@@ -0,0 +1,17 @@
+#' # Define write_csv_fcn() function
+#' 
+#' The `write_csv_fcn()` saves a copy of the processed data in csv format.
+#' 
+#' @param input_data chr, input object (raw data)
```

Comment:
I think instead of `chr`, this would be a `tibble` or `data.frame`, right?

Reply:
Indeed, this is one thing I had to get straight myself today, whether inputs and outputs of the targets are objects, files, etc. I originally had been pulling in the input_directory where the csv was saved, rather than the object from the target. Thanks for the help, I will clean this type of thing up throughout!